src/status_im2/contexts/onboarding/common/background/view.cljs

```diff
@@ -1,26 +1,22 @@
 (ns status-im2.contexts.onboarding.common.background.view
   (:require [react-native.core :as rn]
-            [quo2.foundations.colors :as colors]
-            [status-im2.common.resources :as resources]
-            [react-native.linear-gradient :as linear-gradient]
-            [status-im2.contexts.onboarding.common.background.style :as style]))
+            [react-native.blur :as blur]
+            [status-im2.contexts.onboarding.common.carousel.view :as carousel]
+            [status-im2.contexts.onboarding.common.background.style :as style]
+            [status-im2.contexts.onboarding.common.carousel.animation :as carousel.animation]))
 
 (defn view
   [dark-overlay?]
-  [rn/view
-   {:style style/background-container}
-   [rn/image
-    {:blur-radius (if dark-overlay? 13 0)
-     :style       {:height "100%"
-                   :width  "100%"}
-     ;; Todo - get background image from sub using carousel index on landing page
-     :source      (resources/get-image :onboarding-bg-1)}]
-   [linear-gradient/linear-gradient
-    {:colors [(if dark-overlay? (colors/custom-color :yin 50) "#000716")
-              (if dark-overlay? (colors/custom-color :yin 50 0) "#000716")]
-     :start  {:x 0 :y 0}
-     :end    {:x 0 :y 1}
-     :style  (style/background-gradient-overlay dark-overlay?)}]
-   (when dark-overlay?
+  [:f>
+   (fn []
+     (carousel.animation/initialize-animation)
```

Comment:
why is the carousel here? 🤔  
nearly all of the onboarding pages are using this background component except for the intro page and the intro page is using the carousel component. 

Should we not create a separate background component for this use case?

Reply:
> nearly all of the onboarding pages are using this background component except for the intro page and the intro page is using the carousel component.

Hi, I don't understand what you mean, intro page is also using background component

![image](https://user-images.githubusercontent.com/17097240/228075043-b5ad420e-71b2-4577-92a7-96e98ec81cec.png)

Reply:
ah you are right , it was not before. I understand now the other pages are using this for where the background image has stopped and the section being shown. Nicely done :D

Reply:
Thank you

Reply:
I'd use `useEffect` hook with empty array to do initialization stuff, any reason to not do it?

Reply:
Hi @briansztamfater,
According to the [react docs](https://react.dev/learn/synchronizing-with-effects) - `Effects let you specify side effects that are caused by rendering itself`
So, if I use effect here then the animation will be initialized after rendering.
But I need to initialize it before rendering, otherwise, shared-values will be nil and will throw an error.

Reply:
Just curious, how this will be different then calling function before view?